tokens/properties/alias.json

```diff
@@ -487,6 +490,23 @@
                     "value": "{dimension.dynamic.size.56.value}"
                 }
             }
+        },
+        "spacing": {
```

Comment:
maybe we should change the "padding" property to "spacing" or "layout" instead. to avoid confusion when using "padding" tokens in margins.

Reply:
Spacing tokens (which may after all be called "layout") were the ones devised to be used to define margins, actually. Or maybe I'm not interpreting your comment correctly, sorry!